src/Galaxy/GalaxyAtlas/GalaxyAtlasBuilder.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Hyperdrive\Galaxy\GalaxyAtlas;
+
+use Hyperdrive\Contracts\BuilderContract;
+use Hyperdrive\Galaxy\Geography\Planet;
+use Hyperdrive\Galaxy\Geography\Route;
+use Symfony\Component\Yaml\Yaml;
+
+class GalaxyAtlasBuilder implements BuilderContract
+{
+    protected function __construct()
+    {
+    }
```

Comment:
Czy potrzebujemy tego pustego konstruktora?

Reply:
Tutaj na obronę powiem, że wzięte zostało to stąd: https://github.com/krzysztofrewak/hyperdrive-testing/blob/master/src/GalaxyAtlasBuilder.php i ma swój cel. Ciekawi mnie tylko czy autor @5david5 wie jaki?

Reply:
Klasy takie jak np. `GalaxyAtlasBuilder.php` , `PilotsBuilder.php` posiadają konstruktor z przypisanym modyfikator dostępu `protected` mający na celu uniemożliwić utworzenie instancji danej klasy, ponieważ klasy takie jak `GalaxyAtlasBuilder` miały mieć tylko możliwości wykonywania metod statycznych bez wcześniejszego tworzenia instancji klasy.

Reply:
No tak, nie zauważyłem `protected`. Spoko 👍